### PR TITLE
Admin: Modal dialog

### PR DESCRIPTION
**Pull request by Garfield-fr**
* NEW: Modal dialog component
* NEW: Modal dialog for delete record

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: ui/admin/src/app/core/dialog/dialog.service.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  constructor(private modalService: BsModalService) { }
+
+  public show(config: any) {
+    const bsModalRef = this.modalService.show(DialogComponent, config);
+    return bsModalRef.content.onClose;
+  }
+}
+
+/*
```
Comment (jma) [inline ui/admin/src/app/core/dialog/dialog.service.ts]: to remove?

File: ui/admin/src/app/records/records.service.ts [hunk 2]
```diff
@@ -21,10 +27,13 @@ export class RecordsService {
     );
   }
 
-
-  getRecord(record_type: string, pid: string, resolve = 0) {
```
Comment (jma) [inline ui/admin/src/app/records/records.service.ts]: please leave the resolve param

File: ui/admin/src/app/core/dialog/dialog.service.ts (hunk 1 above)
Comment (Garfield-fr) [reply ui/admin/src/app/core/dialog/dialog.service.ts]: deleted

File: ui/admin/src/app/records/records.service.ts (hunk 2 above)
Comment (Garfield-fr) [reply ui/admin/src/app/records/records.service.ts]: reintroduction of the parameter and update user service

**Pull request merged**